### ISSUE TITLE
Too much .git

### ISSUE THREAD

**Issue by fthomas** [sbt/sbt-github-actions#95]
https://repo1.maven.org/maven2/com/codecommit/sbt-github-actions_2.12_1.0/0.14.2/sbt-github-actions-0.14.2.pom contains this:
```xml
<scm>
<url>
https://github.com/djspiewak/sbt-github-actions.git
</url>
<connection>
scm:git:https://github.com/djspiewak/sbt-github-actions.git.git
</connection>
<developerConnection>
scm:git:git@github.com:djspiewak/sbt-github-actions.git.git
</developerConnection>
</scm>
```
Each URL/connection has an extra `.git` suffix which is the reason why Scala Steward couldn't link to the release notes of 0.14.2. I've no idea where this extra `.git` is coming from.

**Comment by djspiewak** [comment]
Wow… That's really interesting. Sorry! Will fix.

**Comment by fthomas** [comment]
Found the issue: https://github.com/sbt/sbt-git/issues/193. It should be fixed here if [sbt-spiewak-sonatype](https://github.com/djspiewak/sbt-github-actions/blob/228964c555377b6298f907773f126c13124cea10/project/plugins.sbt#L27) is updated to 0.23.0.

**Comment by fthomas** [comment]
This has been fixed in 0.15.0: https://repo1.maven.org/maven2/com/github/sbt/sbt-github-actions_2.12_1.0/0.15.0/sbt-github-actions-0.15.0.pom:
```xml
<scm>
<url>https://github.com/sbt/sbt-github-actions</url>
<connection>scm:git:https://github.com/sbt/sbt-github-actions.git</connection>
<developerConnection>scm:git:git@github.com:sbt/sbt-github-actions.git</developerConnection>
</scm>
```
